starwars/src/components/Items.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './TheLook.css';
+
+function Items(props){
+	const{name,birth_year,gender,eye_color,skin_color,height,mass} = props.character;
```

Comment:
Remember when you know the name of you props you just destructure them 

E.g. ```Items = ({name, birth_year, gender}) => {}```

Then you don't have to manually set all those variables